installer/installer.go

```diff
@@ -33,7 +33,7 @@ type UInstaller interface {
 }
 
 func Install(art io.ReadCloser, dt string, key []byte, scrDir string,
```

Comment:
How about turning this into a struct? say
```
type InstallerConfig struct {
     DeviceType string
     Key *rsa.PublicKey
     ScriptsDir string
     AllowStateScripts bool
}
```
Then the function would become:
`func Install (from io.ReadCloser, device UInstaller, config InstallerConfig) error`, looks cleaner, doesn't it?